### DISCUSSION TITLE
photoprism 230502-c405f6eff

### DISCUSSION THREAD

**Discussion by ippocratis** [Radiokot/photoprism-android-client#37] [Q&A] [answered]
photoprism 230502-c405f6eff is out
Anyone tried the app with the latest version?
Breaks anything? 

Thanks

**Comment by leosamuele221** [comment, marked as answer]
Just updated.
Seems to work normally

**Reply by ippocratis** [reply]
Thanks for the feedback
I'll update my instance too

**Comment by Radiokot** [comment]
It works fine on the demo, though I haven't tested the login. Waiting for armv7 docker image 🙄

**Comment by leosamuele221** [comment]
Login seems to work to, just tried it

**Reply by Radiokot** [reply]
Thank you Samuele
